### DISCUSSION TITLE
Unavailable To Borrow

### DISCUSSION THREAD

**Discussion by youperb** [elementdavv/internet_archive_downloader#45] [Q&A]
Internet Archive has suddenly disabled more than 500.000 books which were available to borrow a few weeks ago. Those books are now marked as 'Borrow Unavailable '. Is there any possiblity to download these books.
Here is an example:
https://archive.org/details/ladybirdcoversto0000unse

**Comment by elementdavv** [comment]
Duplicate of #49.
